index.js

```diff
@@ -1,18 +1,40 @@
-/***** Deliverable 1 *****/
-const header = document.querySelector("#header")
-console.log("Here's your header:", header)
 
+//D1
+const header = document.querySelector("#header")
+console.log("Here's your header: ", header)
 
-/***** Deliverable 2 *****/
+//D2
+header.style.color = "green"
 
+//D3
+let travelerObj = document.querySelector(".traveler #profile")
+console.log("this is what the traveler obj looks like", travelerObj)
 
+let h2 = document.querySelector("#profile .name")
+h2.textContent = traveler.name
+console.log(h2)
 
-/***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+let em = document.querySelector("#profile .nickname")
+em.textContent = traveler.nickname
 
+let img = document.querySelector("#profile .photo")
+img.src = traveler.photo
+img.alt = traveler.name
 
-/***** Deliverable 4 *****/
+//D4
+const allAnimalSightings = traveler.animalSightings
+let animalSightingList = document.querySelector('#animals')
```

Comment:
Nice work for this too! In this case `traveler` would also be `travelerObj` again :)